content/docs/alerting/notifications.md

```diff
@@ -86,3 +86,4 @@ templating.
 | toLower | string | [strings.ToLower](http://golang.org/pkg/strings/#ToLower), converts all characters to lower case. |
 | safeHtml | text string | [html/template.HTML](https://golang.org/pkg/html/template/#HTML), Marks string as HTML not requiring auto-escaping. |
 |reReplaceAll | pattern, replacement, text | [Regexp.ReplaceAllString](http://golang.org/pkg/regexp/#Regexp.ReplaceAllString) Regexp substitution, unanchored. |
+| match | pattern, string | [Regexp.MatchString](https://golang.org/pkg/regexp/#MatchString). Match a string using Regexp. |
```

Comment:
Can we keep the ordering the same as for https://prometheus.io/docs/prometheus/latest/configuration/template_reference/#strings?

Reply:
yes.

Reply:
I also moved toUpper, toLower to the same place as they're at the prometheus docs.